frontend/src/components/NftCard.js

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+// SPDX-FileCopyrightText: 2021 Dominic Heil <d.heil@campus.tu-berlin.de>
+
+import React from "react";
+import HeaderTypography from "./HeaderTypography";
+import ParagraphTypography from "./ParagraphTypography";
+import RoundedEdgesButton from "./RoundedEdgesButton";
+import EthereumIconSvg from "../assets/img/ethereumIcon.svg";
+import { Link } from "react-router-dom";
+import * as assert from "assert";
+
+const EthereumIcon = () => <img src={EthereumIconSvg} alt="ETH" style={{ marginBottom: "-4px" }}/>;
+
+export default function NftCard({ title="",
+	description="",
+	price="-",
+	nftPageUrl="",
+	imgUrl="	",
+	buttonText="Access",
+	sm=false,
+	md=false,
+	lg=true,
```

Comment:
No, it is not. I defined them similar to sm, md, lg in MUI https://mui.com/api/grid/. If you create a component like `<NftCard sm .../>` then it will be a small card, `md` for a medium sized card and `lg` for a large card.

Reply:
`enum` means, that they are supposed to be exclusive, i.e. they should never be true at the same time. It looks like this is the case. In this case, an `enum` would be the better way to express this.

Reply:
Changed it to an enum.